src/v1/Controllers/Token.php

```diff
@@ -165,6 +165,17 @@ public function postToken(Request $request, Response $response, $args): Response
     $future = new DateTime("+20 minutes");
     // For test / DEBUG
     // $future = new DateTime("+30 seconds");
+    // Get roles
+    $roleId = null;
+    foreach ($user->roles()->get() as $role)
+    {
+      $roleId = $role->id;
+      break;
+    }
```

Comment:
You really like loops :p

```suggestion
    $role = $user->roles()->first();
```

Or simply (if you go with my suggestion to rename `roles()` to `role()`):

```suggestion
    $role = $user->role();
```

Then check if `$role` is null line 175 and call `$role->id` line 187.

Reply:
Fixed